js/data/bucket/fill_bucket.js

```diff
@@ -15,7 +16,7 @@ const fillInterfaces = {
             components: 2,
             type: 'Int16'
         }]),
-        elementArrayType: new ElementArrayType(1),
+        elementArrayType: new ElementArrayType(3),
```

Comment:
`new ElementArrayType(3)` looks correct to me. I wonder why this was `1` before.